### DISCUSSION TITLE
OpenAI Whisper -- A minor error?

### DISCUSSION THREAD

**Discussion by Mattral** [openai/whisper#2027] [Q&A]
I decided to install a Whisper to transcribe my recent interviews for my current study

Hopefully I did it well, but now Whisper gets me an error every time I launch it:

[in#0 @ 0000015a2360ed80] Error opening input: No such file or directory
Error opening input file first_audio.mp3.
Error opening input files: No such file or directory
Surprisingly, I do have to load an mp3 so what Whisper can find it. But where exactly should I? Or maybe this can be fixed in a different way.

I used the same code from GitHub:

```
import whisper

model = whisper.load_model("base")

result = model.transcribe("audio7.mp3")

print(result["text"])
```

I tried literally to upload an audio from my desktop to the Whisper's directory, but I failed

**Comment by phineas-pta** [comment]
it's more like u having problem navigating files & folders
